docs/integration_fetching_data.md

```diff
@@ -49,19 +49,19 @@ async def create_coordinator(api):
         hass,
         _LOGGER,
         # Name of the data. For logging purposes.
-        "sensor",
-        async_update_data,
+        name="sensor",
+        update_method=async_update_data,
         # Polling interval. Will only be polled if there are subscribers.
-        timedelta(seconds=30),
+        update_interval=timedelta(seconds=30),
         # Debouncer to limit request_refresh
-        debounce.Debouncer(
+        request_refresh_debouncer=debounce.Debouncer(
```

Comment:
Is this parameter worth mentioning in the example case? Looks like there is a default debouncer with some sane defaults already in place, so perhaps just mention it in the text for those special cases where adjustments are necessary?